TestODESolvers/test/test-rkf78.par

```diff
@@ -17,6 +17,7 @@ CarpetX::blocking_factor_z = 1
 
 CarpetX::ghost_size = 0
 
+Driver::periodic = "yes"
 CarpetX::periodic_x = yes
```

Comment:
Shouldn't these parameters be renamed to `Driver::periodic_x` etc.?

Reply:
should be , yes.

Reply:
That's in the output directory. The par file should not even be there. Removed.